### PR TITLE
feat: update post to match new route format

### PR DESCRIPTION
**Pull request by thlmenezes**
## Description of change

Updates post routing to match new proposed format to generate help links in web app

## Issues resolved by this PR

- [x] [DVN-11614]
- [x] [DVN-11637]
- [x] [DVN-11638]
- [x] [DVN-11639]
- [x] [DVN-11640]
- [x] [DVN-11641]
- [x] [DVN-11642]
- [x] [DVN-11643]
- [x] [DVN-11644]
- [x] [DVN-11645]
- [x] [DVN-11646]
- [x] [DVN-11647]

## More info

[DVN-11614]: https://devopness.atlassian.net/browse/DVN-11614?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DVN-11637]: https://devopness.atlassian.net/browse/DVN-11637?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DVN-11638]: https://devopness.atlassian.net/browse/DVN-11638?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DVN-11639]: https://devopness.atlassian.net/browse/DVN-11639?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DVN-11640]: https://devopness.atlassian.net/browse/DVN-11640?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DVN-11641]: https://devopness.atlassian.net/browse/DVN-11641?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DVN-11642]: https://devopness.atlassian.net/browse/DVN-11642?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DVN-11643]: https://devopness.atlassian.net/browse/DVN-11643?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DVN-11644]: https://devopness.atlassian.net/browse/DVN-11644?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DVN-11645]: https://devopness.atlassian.net/browse/DVN-11645?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DVN-11646]: https://devopness.atlassian.net/browse/DVN-11646?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DVN-11647]: https://devopness.atlassian.net/browse/DVN-11647?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by thlmenezes** [comment]
Deployed pipeline for `PR #728 (feat: update post to match new route format)`: Devopness application `thlmenezes-site.devopness.com` deployment **in-progress** on action <https://thlmenezes-site.devopness.com>

**Comment by thlmenezes** [comment]
Deployed pipeline for `PR #728 (feat: update post to match new route format)`: Devopness application `thlmenezes-site.devopness.com` deployment **completed** on action <https://thlmenezes-site.devopness.com>

**Comment by thlmenezes** [comment]
Deployed pipeline for `PR #728 (feat: update post to match new route format)`: Devopness application `thlmenezes-site.devopness.com` deployment **in-progress** on action <https://thlmenezes-site.devopness.com>

**Comment by thlmenezes** [comment]
Deployed pipeline for `PR #728 (feat: update post to match new route format)`: Devopness application `thlmenezes-site.devopness.com` deployment **completed** on action <https://thlmenezes-site.devopness.com>

File: docs/environments/team-memberships/index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: Memberships
```
Comment (plantaz) [inline docs/environments/team-memberships/index.md:2]:
```suggestion
title: Environment Team Memberships
```

File: docs/pipelines/steps/index.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+title: Steps
```
Comment (plantaz) [inline docs/pipelines/steps/index.md:2]:
```suggestion
title: Pipeline Steps
```

File: docs/ssl-certificates/index.md
Comment (plantaz) [inline docs/ssl-certificates/index.md:1]: Teremos algum redirecionamento caso o usuário tente acessar a partir de `docs/applications/ssl-certificates/*`?

Pq imagino que no web-app estaríamos puxando a partir desse path, certo?

File: docs/teams/members/index.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+title: Members
```
Comment (plantaz) [inline docs/teams/members/index.md:2]:
```suggestion
title: Team Members
```

**Review by plantaz — APPROVED** [review]
LGTM

(Some threads to discuss, but I imagine these are points to be defined going forward rather than being resolved in this PR. If you agree, they could become tasks!)

File: docs/ssl-certificates/index.md
Comment (thlmenezes) [reply docs/ssl-certificates/index.md:1]: @plantaz in the web-app help popover, the user is redirected to https://www.devopness.com/ssl-certificates/view-ssl-certificate from routes such as `projects/:id/environments/:id/applications/:id/ssl-certificates/:id`; only selected cases use the parentModel

**Comment by thlmenezes** [comment]
> Algumas thread para discutirmos mas imagino que sejam pontos a serem definidos daqui para frente do que de serem resolvidos nessa PR e caso concordem poderiam virar tasks!

I believe those discussions can be made, with its respective links, in the project channel; as this is a major updated that solves many tasks that would roll over to another sprint, I'm closing discussions and merging this PR.

Let's keep improving the documentation together 😄

**Pull request merged**